lessons/day_02/java/src/main/resources/application.properties

```
@@ -4,7 +4,4 @@ logging.level.org.hibernate=ERROR
 spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
 
 spring.h2.console.enabled=true
-spring.datasource.url=jdbc:h2:mem:graphQL
 spring.datasource.name=graphQL
```

No necesitamos tambien tener las properties para graphiql?
```suggestion
spring.datasource.name=graphQL

spring.graphql.graphiql.enabled=true
spring.graphql.graphiql.path=/playground
```